### ISSUE TITLE
Комментарии к заявке

### ISSUE THREAD

**Issue by aakuptsov** [GeorgFed/On-My-Way#1]
1. Файл с заявкой должен быть назван _фамилия_application.md_

**Comment by GeorgFed** [comment]
> 1. Является ли это приложение только интегратором функционала, который предоставляется отдельными приложениями, или в нём есть что-то такое, чего нет в органайзерах, планировщиках, приложениях по бюджетированию личных финансов и т. п.?

Да, подобный функционал представлен во многих приложениях, однако особенность On My Way заключается в объединении этих функций в одном приложении. Приложение выполняет роль удобного ассистента, чтобы пользователю не приходилось сначала создавать расписание дня в одном приложении, а потом искать каждое из мест, которые ему надо посетить, в другом. При этом оно заранее напоминает о том, что пора выезжать и показывает к маршрут, на котором находится пользователь сейчас, а также еще и предстоящий путь на этот день.
